src/test/java/integration/TypeOptionsTest.java

```diff
@@ -0,0 +1,32 @@
+package integration;
+
+import com.codeborne.selenide.TypeOptions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.Duration;
+
+import static com.codeborne.selenide.Condition.attribute;
+
+class TypeOptionsTest extends ITest {
+  @BeforeEach
+  void openAndInitTestPage() {
+    openFile("page_with_inputevents_on_clear.html");
+  }
+
+  @Test
+  void typeOptions() {
+    $("#username_input").setValue("test");
```

Comment:
Perhaps you should separate these tests for clarity of the results of the checks.

Reply:
@adorne  - thanks for the suggestion. Now the test is short and we don't need to separate this. 

@asolntsev - Please review again as I have already pushed all the changes